app/services/scholars_archive/google_scholar_citation_service.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module ScholarsArchive
+  # Class to handle the expected format for citation publication dates as
+  # required in the indexing guidelines for configuring meta-tags:
+  # https://scholar.google.com/intl/en/scholar/inclusion.html#indexing
+  class GoogleScholarCitationService
+    attr_reader :input_date
+
+    def self.call(input_date)
+      new(input_date).call
```

Comment:
I think you could cut out the extra #call here. `new(input_date).format_date` would work peachy

Reply:
good call